### PR TITLE
Add unit test for AutoPauseTrigger

### PR DESCRIPTION
**Pull request by nebmo**

### CODE REVIEW

**Comment by jonasoreland** [comment]
looks great!
comment:
1) should travis to actually run the tests too (i didn't see that it did)

**Comment by nebmo** [comment]
Yeah you are correct! I dont know how though... do you know?

**Comment by jonasoreland** [comment]
no idea...but the travis.yml doesn't look to complicated...so it shouldn't
be that hard

/Jonas

**Comment by nebmo** [comment]
I dont understand, froyo is not building for me:
runnerup\common\src\test\java\org\runnerup\common\util\ValueModelTest.java
Error:(19, 13) error: cannot find symbol class ValueModel
Error:(30, 19) error: package ValueModel does not exist

**Comment by nebmo** [comment]
Changed so it only runs the tests for Latest. Guess that have to be enough for now..  you can merge

**Comment by jonasoreland** [comment]
guess
- when David added the ValueModelTest there where no automatic testing.
- hence they have not been run since
- hence they have rotten

wdyt ?

/Jonas

File: app/build.gradle [hunk 1]
```diff
@@ -61,6 +62,7 @@ android {
         // but continue the build even when errors are found:
         fatal 'NewApi', 'InlineApi'
         disable 'MissingTranslation'
+        abortOnError false
```
Comment (jonasoreland) [inline app/build.gradle]: what does this mean ? why is this change needed ?

Comment (nebmo) [inline app/build.gradle]: We get lint errors so it would break the build otherwise. Wasn't there another PR that was about lint errors?

**Comment by nebmo** [comment]
I really dont know what is wrong with ValueModelTest. AS gives no clue...

**Comment by nebmo** [comment]
Same problem here I think:
http://stackoverflow.com/questions/28782518/android-unit-testing-not-working-for-android-library-project

**Comment by jonasoreland** [comment]
i rather have you disable the ValueModelTest then add "abortOnError false"

**Comment by jonasoreland** [comment]
(or best would ofcourse be to fix it...)
but disable it is also fine...

**Comment by nebmo** [comment]
Updated build tools and renamed a language folder (copied from other PR). now it works

**Comment by jonasoreland** [comment]
great!

**Comment by nebmo** [comment]
So.. next step is to get some database rows from a workout where autopause didnt work. you have?
